README.md

```diff
@@ -58,16 +58,30 @@ To look up a document the *docId* is needed. Once a *docId* is known a node can
 
 In Ceramic, each document has a unique identifier (docId). The identifier is a string with the following format:
 
+```
+ceramic://<CID-of-genesis-record>
+```
+
+Sometimes Ceramic identifiers might also be formated as:
+
 ```
 /ceramic/<CID-of-genesis-record>
 ```
 
-For example, a docId might look like this:
+As an example, a docId might look like this:
 
 ```
-/ceramic/bafyreihl3rizxqjkedmp7rdckrqd3kufwe5e7c6xejcoheqo7rp63idsva
+ceramic://bafyreihl3rizxqjkedmp7rdckrqd3kufwe5e7c6xejcoheqo7rp63idsva
 ```
 
+### Document versions
+Each time a document is [anchored on a blockchain](#blockchain-anchoring) with an *anchor record*, a new version of the document is created. Each version of the document can be refered to by using the following format:
+
+```
+ceramic://<CID-of-genesis-record>?version=<CID-of-anchor-record>
```

Comment:
Are we now just using this format and not also numerical versions?

Reply:
Yeah, no numerical versions for now. This way is more secure.